data_pipeline_api/pipeline.py

```diff
@@ -1,12 +1,58 @@
 import datetime
+import json
 import logging
 import os
 
+import requests
 import yaml
 
 from data_pipeline_api import fdp_utils
 
 
+def search(
+    handle: dict,
+    token: str,
```

Comment:
Token should be removed or optional on get requests, the only table which require the token in a get request is user and groups.